web/src/tippyjs.ts

```diff
@@ -5,6 +5,7 @@ import tippy, {delegate} from "tippy.js";
 import render_tooltip_templates from "../templates/tooltip_templates.hbs";
 
 import {$t} from "./i18n";
+import {hide_all} from "./popovers";
 import {user_settings} from "./user_settings";
```

Comment:
I know the two other imports in this file work differently, but please import this as `import * as popovers from "./popovers";` and access it as `popovers.hide_all`. Those two other modules are special; 98% of modules should be imported the `*` way so the namespacing is clear.

Reply:
Updated . Hope that works .